commands.js

```diff
@@ -204,14 +205,17 @@ function formatSummary ( data, favoriteMode ) {
   if (data.playing) {
     colorEmoji = data.playing.includes("white") ? "⚪" : "⚫";
   }
-  console.log(colorEmoji);
-  var status = ( !data.online ? " 🔴Offline" : ( colorEmoji ? colorEmoji + "Playing" : " ✅Online" ) )
-  console.log(status);
+  var status = ( !data.online ? " 🔴Offline" : ( colorEmoji ? colorEmoji + "Playing" : " ✅Online" ) );
+
+  var flag = "";
+  if (data.profile.country)
+    flag = " " + countryFlags.countryCode(data.profile.country).emoji;
+
   var formattedMessage;
 	formattedMessage =
 		data.url + "\n" +
 		"```prolog\n" +
-		"User: " + data.username + getHighestRating( data.perfs ) + "\n"+
+		"User: " + data.username + flag + getHighestRating( data.perfs ) + "\n"+
```

Comment:
Could we maybe move Offline / Online to the end of this line?  Every Lichess page displays username and online status on the same line.